hbase-server/src/test/java/org/apache/hadoop/hbase/replication/TestRemoveFromSerialReplicationPeer.java

```diff
@@ -89,14 +93,18 @@ public void testRemoveTable() throws Exception {
     waitUntilHasLastPushedSequenceId(region);
 
     UTIL.getAdmin().updateReplicationPeerConfig(PEER_ID,
-      ReplicationPeerConfig.newBuilder(peerConfig).setTableCFsMap(Collections.emptyMap()).build());
+      ReplicationPeerConfig.newBuilder(peerConfig)
+        .setTableCFsMap(Collections.emptyMap())
+        .setReplicateAllUserTables(true)
+        .setExcludeTableCFsMap(ImmutableMap.of(tableName, Collections.emptyList())).build());
 
```

Comment:
The previous test behaviour isn't valid anymore, as it was relying on an empty `namespaces` collection to filter out entries, but now, passing null or empty `namespaces` just activates the scoped filter, which will cause these table entries to be picked for replication, as its scope is set to `1`.